output/forms.eslint.txt

```diff
@@ -2,63 +2,21 @@
 > react-talend-forms@0.111.0 lint:es /home/travis/build/Talend/ui/packages/forms
 > eslint --config .eslintrc src
 
-The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
-
-/home/travis/build/Talend/ui/packages/forms/src/fields/ArrayField.js
```

Comment:
ignore some files to avoid to get diff with the original files (react-jsonschema-format)